.github/workflows/deploy-fleet-website.yml

```diff
@@ -8,8 +8,13 @@ on:
       - 'docs/**'
       - 'handbook/**'
       
+permissions:
+  contents: read
+
 jobs:
   build:
+    permissions:
+      contents: write  # for Git to git push website
```

Comment:
Is that needed for a push to Heroku? I didn't think that a website deploy would need write access.

Reply:
You're probably right - I was looking at jobs and saw git push and my brain segfaulted 😅 I will go remove that section.